tests/perf/llama/run.c

```diff
@@ -0,0 +1,941 @@
+/* Inference for Llama-2 Transformer model in pure C */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <time.h>
+#include <math.h>
+#include <string.h>
+#include <fcntl.h>
+#if defined _WIN32
+    #include "win.h"
+#else
```

Comment:
remove it?

Reply:
Just following the upstream but totally ok to remove

https://github.com/karpathy/llama2.c/blob/master/run.c#L10-L12